src/core/components/configuration.ts

```diff
@@ -82,6 +83,25 @@ export const makeConfiguration = (
   base: ExtendedConfiguration & PlatformConfiguration,
   setupCryptoModule?: SetupCryptoModule,
 ): PrivateClientConfiguration & PrivateConfigurationFields => {
+  // Set default retry policy for subscribe (if new subscribe logic not used).
+  if (!base.retryConfiguration && base.enableEventEngine) {
+    base.retryConfiguration = RetryPolicy.ExponentialRetryPolicy({
+      minimumDelay: 2,
+      maximumDelay: 150,
+      maximumRetry: 6,
+      excluded: [
+        Endpoint.MessageSend,
```

Comment:
Wondering, should we use publish instead?

Reply:
Also it can be confusing with signal/fire.

Reply:
Same naming picked for the Kotlin. 
General group is for messaging (signal is just short form of message).

Reply:
@mohitpubnub I had this one for Objective-C and idea was that all of that is a messages (signal is size constraint message) and there is no `fire` (it will be sunsetted sooner or later). 
Also I tried to keep name kind of consistent with message storage (but it is different for Kotlin and called message persistence).

You think it is this bad? Kotlin has `Publish` - this also will conflict with `signal`.

Reply:
Swift and Objective-C use almost similar names

Reply:
It's fine. I agree that `publish` can be confusing and do not mix with signal.